herd/tests/instructions/AArch64.mixed/A14.litmus

```diff
@@ -0,0 +1,9 @@
+AArch64 A14
+(* Store register, with offset, symbolic location *)
+
+ { uint64_t x; 0:x0=1; 0:x2=x; 0:x3=4;}
```

Comment:
Declaring register 0:x3 to be used as a 64bits register as follows would avoid a (litmus) warnig.
```suggestion
 { uint64_t x; 0:x0=1; 0:x2=x; int64_t 0:x3=4;}
```